src/__fixtures__/github-schema.json

```diff
@@ -56052,7 +56052,7 @@
             },
             {
               "name": "INTERFACE",
-              "description": "Indicates this type is an interface. `fields` and `possibleTypes` are valid fields.",
+              "description": "Indicates this type is an interface. `fields`, `interfaces`, and `possibleTypes` are valid fields.",
```

Comment:
It's introspection snapshot of public GitHub API that we use for perf testing.
Please revert.

Reply:
Hmm, should this be reverted even though [the source line](https://github.com/mike-marcacci/graphql-js/blob/6f85d6b33c20c0de6556d153cd723dac4324472a/src/type/introspection.js#L401) has changed?

Reply:
@mike-marcacci It's literally the result of introspection query against GitHub public API. Next time someone updates it this change will be lost.

Reply:
Ah, gotcha. Updated!